3ds Max/Max2Babylon/Exporter/BabylonExporter.cs

```diff
@@ -476,7 +476,12 @@ public void Export(MaxExportParameters exportParameters)
                 BabylonNode node = exportNodeRec(maxRootNode, babylonScene, gameScene);
                 // if we're exporting from a specific node, reset the pivot to {0,0,0}
                 if (node != null && exportNode != null && !exportNode.IsRootNode)
-                    SetNodePosition(ref node, ref babylonScene, new float[] { 0, 0, 0 });
+                {
+                    if (!exportParameters.exportKeepNodePosition)
```

Comment:
Why did the behavior differ between multi-export and regular export?

Reply:
This is THE question.. I do not know. It's not obvious to see the meaning behind these lines. Additionaly, another question is why there is two differents path (differents properties into the exporter ) to export single or multi-object.. Here, my proposal is only about a fix to give user a better experience without excessive changes. 
In term of software, i'm agree, it's ugly 🥇 .

Reply:
hmm, I think it may be best to understand why the behavior differs, instead of making a spot fix here. If we're going down separate codepaths for the different export methods, we should probably figure out how to combine those paths, I was under the impression that multiexport was merely a batching of export commands, and shouldnt change behavior...

Reply:
I already conduct this research and did not find any informations into the code. It's like this is a quick and dirty response to a feature request... The only noticiable diff is about the export parameters augmented with a node property, which lead to trigger these lines.
We may have to search down the PR history to see when it has been introduced.